GramAddict/core/utils.py

```diff
@@ -243,5 +243,39 @@ def validate_url(x):
         return False
 
 
+def read_file(filename):
+    if not filename.lower().endswith(".txt"):
+        filename = f"{filename}.txt"
+    if os.path.isfile(filename):
+        with open(filename, "r") as f:
+            return f.readlines()
+    else:
+        return False
+
+
+def delete_line_in_file(line_to_delete, filename):
```

Comment:
This is really inefficient. We need to do this better. Every time we need to delete a line, we read through the whole file to find it? It would make more sense to re-write the open file when we are done running. We just have to be careful to catch crashes somehow

Reply:
oke, sounds good to me.. I'll think it better tomorrow